#1271 Updates setup links and example GKE version

PatrickAuld opened the pull request:
Adds a setup link for AWS and updates the Openshift and IKS links to go directly to tutorial pages for those systems. Before these linked to the marketing pages. Updates the GKE example to use the latest Kubernetes version.

istio-testing commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: 
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **linsun**

Assign the PR to them by writing `/assign @linsun` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/istio/istio.github.io/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["linsun"]} -->

istio-testing commented:
Hi @PatrickAuld. Thanks for your PR.

I'm waiting for a [istio](https://github.com/orgs/istio/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/istio/people) to skip this step.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/devel/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

gargnupur commented on _docs/setup/kubernetes/quick-start.md:
```diff
@@ -93,7 +93,7 @@ $(bx cs cluster-config <cluster-name>|grep "export KUBECONFIG")
 Configure `kubectl` CLI based on steps [here](https://www.ibm.com/support/knowledgecenter/SSBS6K_2.1.0/manage_cluster/cfc_cli.html) for how to access the IBM Cloud Private Cluster.
 
-### [OpenShift Origin](https://www.openshift.org) (version 3.7 or later)
+### [OpenShift Origin](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) (version 3.7 or later)
```
/cc @hklai 

Should these links be added separately as:
Follow steps here(https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) to install kubernetes on OpenShift Cloud.

gargnupur commented on _docs/setup/kubernetes/quick-start.md:
```diff
@@ -110,7 +110,7 @@ Service account that runs application pods need privileged security context cons
+### [AWS (w/Kops)](https://aws.amazon.com/blogs/compute/kubernetes-clusters-aws-kops/)
 
 When you install a new cluster with Kubernetes version 1.9, prerequisite for `admissionregistration.k8s.io/v1beta1` enabled is covered.
 
```
Can we add a link for Azure too?

istio-testing commented:
@PatrickAuld: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

geeknoid commented:
Do we want to proceed with this PR, or should we close it?

gargnupur commented:
I think this is still needed...

looks like gke version specified is 1.9.7-gke.1 which is not a valid version in https://cloud.google.com/kubernetes-engine/versioning-and-upgrades

istio-testing commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/istio/istio.github.io/pull/1291#" title="Author self-approved">PatrickAuld</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **geeknoid**

Assign the PR to them by writing `/assign @geeknoid` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/istio/istio.github.io/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["geeknoid"]} -->

PatrickAuld commented:
Apologies for the delay. I've removed the links from the headers and added a line at the start of each section with a link to docs, expect for the Google and IBM Private Cloud ones as the former's instructions are inlined and the latter's docs don't appear to be public.

geeknoid commented:
Deploy preview for *preliminary-istio* failed.

Built with commit b5728cb5c170da16c9f95e29d533d62ee1683454

https://app.netlify.com/sites/preliminary-istio/deploys/5b1f06b9b13fb17f39bdf093

geeknoid commented:
@hklai @gargnupur Please review.

hklai commented on content/docs/setup/kubernetes/quick-start.md:
````diff
@@ -76,7 +76,9 @@ $ kubectl create clusterrolebinding cluster-admin-binding \
 ```
 
-### IBM Cloud Kubernetes Service (IKS)
+### IBM Cloud Container Service (IKS)
````
isn't the name 'IBM Cloud Kubernetes Service' instead?

istio-testing commented:
@PatrickAuld: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

geeknoid commented:
@PatrickAuld Do you want to address the comments?
